front/src/components/Dialogs/SearchDialog.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <q-dialog ref="dialogRef" @hide="onDialogHide">
+    <q-card style="min-width: 80%">
+      <q-card-section class="text-h6"> Search for CTF/Task </q-card-section>
+      <q-separator />
+      <q-card-section>
+        <q-input
+          v-model="searchText"
+          filled
+          label="What are you searching for?"
+          hint="Search for title or description"
```

Comment:
Can you change the hint to `Search for title of CTF or task`? Description is not searchable.